### DISCUSSION TITLE
Is it possible to use dpgen with a pretrained model as a start?

### DISCUSSION THREAD

**Discussion by luanhw** [deepmodeling/dpgen#1262] [General]
Hi,

I'm a new user in dpgen. I have noticed that the dpgen usually starts training the deep learning network with DFT data. I wonder if it is possible to use a pretrained model as a start, and then we first fine tuning the pretrained model with DFT data, and then we continue the regular dpgen process.

Thank you,
Hengwei LUAN

**Comment by njzjz** [comment]
No. It has been listed as an issue in #1122. You are welcome to contribute the code.
